.azure/pipelines/jobs/default-build.yml

```diff
@@ -116,6 +116,10 @@ jobs:
     - DOTNET_CLI_HOME: $(System.DefaultWorkingDirectory)
     - DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
     - TeamName: AspNetCore
+    - ${{ if eq(parameters.agentOs, 'Linux') }}:
```

Comment:
nit: Up to you but I wouldn't bother w/ making this platform-specific. The variables can help on macOS and don't hurt on Windows.